File: JAVA/Array.java
import java.util.*;

class Array
{
    public static void main(String Arg[])
    {
        Scanner sobj = new Scanner (System.in);

        int isize = 0, i = 0;
        System.out.println("Enter the number of elememts :");
        isize = sobj.nextInt();

        int Arr[] = new int[isize];

        System.out.println("Enter Elements in array");
        for(i = 0; i < isize; i++)
        {
            Arr[i]= sobj.nextInt();
        }
        System.out.println(" Elements in array are");
        for(i = 0; i < isize; i++)
        {
            System.out.print(Arr[i]+"\t");
        }
        System.out.println();
        
        sobj.close();
    }
}

File: C++/Array.cpp
#include<iostream>
using namespace std;

class Array
{
    private:
        int *Arr;      //int  Arr[];
        int iSize;

    public:
        Array(int x)
        {
            iSize = x;
            Arr = new int[iSize];
        }
        ~Array()
        {
            delete []Arr;
        }
        void Accept()
        {
            int iCnt = 0;
            cout<<"Please enter the elements :"<<endl;
            for(iCnt = 0; iCnt < iSize; iCnt++)
            {
                cin>>Arr[iCnt];
            }

        }
        void Display()
        {
            cout<<"Elements in Array are :"<<endl;
            int iCnt = 0;
            for(iCnt = 0; iCnt < iSize; iCnt++)
            {
                cout<<Arr[iCnt]<<"\t";
            }
            cout<<endl;
        }
};

int main()
{
    int iValue = 0;

    cout<<"Enter the elements Want to store :"<<endl;
    cin>>iValue;

    Array *aobj = new Array(iValue);
    aobj->Accept();
    aobj->Display();

    delete []aobj;  //it will first ask compiler does it have destructor if yes then it 
                    // will first call destructor and then deallocate the dynamically 
                    //allocated memmory...

    return 0;
}